source/exercicios_controle.rst

```diff
@@ -19,10 +49,42 @@
    possui duas raízes reais; se delta é 0, possui uma raiz; caso delta seja
    menor que 0, não possui raiz real
 
+   .. only:: instructors
+
+      Exemplo de solução:
+
+      .. code-block:: python3
+
+         >>> a = int(input("Qual o valor de a? "))
+         >>> b = int(input("Qual o valor de b? "))
+         >>> c = int(input("Qual o valor de c? "))
+         >>> delta = b**2 - 4*a*c
+         >>> if delta > 0:
+         ...   print("A equação possui duas raizes")
+         ... elif delta == 0:
```

Comment:
Comparar floats _normalmente_ tráz problemas pelo arredondamento/truncamento que é feito. O que fazemos nesses casos é comparar a diferença entre os números e ver se é praticamente zero. Algo como `if abs(a - b) < 1e-10`. Para esse exercício, o modo que você fez resolve. Se um dia você for trabalhar com cálculo numérico, lembra desse detalhe.

A Wikipedia tem uma seção sobre o problema de precisão dos floats: https://en.wikipedia.org/wiki/Floating-point_arithmetic#Minimizing_the_effect_of_accuracy_problems